client/src/pages/EventsPage/index.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import './style.css'
+import FormControl from "react-bootstrap/FormControl";
+import InputGroup from "react-bootstrap/InputGroup";
+import Button from "react-bootstrap/Button";
+import {ButtonToolbar, CardDeck, Container, Row} from "react-bootstrap";
+import DropdownButton from "react-bootstrap/DropdownButton";
+import {Dropdown} from "react-bootstrap";
+import EventCard from "./event-card";
+import Col from "react-bootstrap/Col";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+
+class EventsPage extends React.Component{
+
+  render (){
+    const eventsList = [<EventCard />, <EventCard />,<EventCard />, <EventCard />, <EventCard />, <EventCard />, <EventCard />];
+    const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+    const eventsType = ['Party', 'Karaoke', 'Concert', 'For children', 'Master class', 'Tasting', 'Sports broadcasting'];
+
+    const CustomDatePicker = ({ value, onClick }) => (
+      <Button variant="warning" className="example-custom-input" onClick={onClick}>
+        {value} Date Select
+      </Button>
+    );
+
+    return (
+     <Container fluid className="page-container p-0">
+
+       <Container fluid className="page-header">
+         <h1 className="page-header-title"> All events and activities in restaurants, <br/> cafes and bars</h1>
+          <Container className="search-container pt-4">
+          <InputGroup className="mb-3">
+            <FormControl
+              placeholder="Search events"
+              aria-label="Search events"
+              aria-describedby="basic-addon2"
+            />
+            <InputGroup.Append>
+              <Button className="button">Find</Button>
+            </InputGroup.Append>
+          </InputGroup>
+          </Container>
+
+         <Container fluid className="filter-and-sort">
+           <ButtonToolbar variant="warning" className="justify-content-center pb-4 pt-4">
+             {/* Sort by type */}
```

Comment:
Could you remove comments? Your called variable `eventsType`. It is selfdocumented name